internal/shared/person.go

```diff
@@ -9,6 +11,10 @@ type Address struct {
 	Country  string `json:"country"`
 }
 
+func (a Address) IsSet() bool {
```

Comment:
This is such a nit, but might be clearer to define `IsZero()` and check the inverse since "zero" already has clear meaning in Go (arguably `person.Address = Address{}` would be "set" even though it's empty).

Reply:
cool, and it matches `time.Time` then which is nice